docs/src/main/asciidoc/build-analytics.adoc

```diff
@@ -0,0 +1,156 @@
+
+== This is the data being collected
+
+* Anonymous application name. SHA-256 hash of the Group Id + Artifact Id.
+* Anonymous application version. SHA-256 hash of the Artifact Id + Version.
+* Anonymous user id. During the 1st build, this is generated locally at random, and subsequently reused.
+* Quarkus version
+* Java version
+* GraalVM version
+* Operating system details
+* Build system details (Maven, Gradle, etc)
+* If a CI system was detected
+* Country and timezone
+* Extensions enabled. Only extensions with grouIds starting with `io.quarkus` or `io.quarkiverse` are collected.
+
+And here is an example of the data payload being sent:
+
+[source,json]
+.Example analytics payload
+----
+{
+  "context": {
+    "app": {
+      "name": "ZzOlt3q0VgNK0oyUd6NDAhPqJjTSxNOAnqHb5dHfS7M=",
+      "version": "FUDdh16tTKtuJkcoY2Wp57IKKwxZ/Hx42tTvi3Izpmc="
+    },
+    "build": {
+      "gradle_version": "N/A",
+      "maven_version": "3.8.7"
+    },
+    "ci": {
+      "name": "unknown"
+    },
+    "graalvm": {
+      "java_version": "N/A",
+      "vendor": "N/A",
+      "version": "N/A"
+    },
+    "ip": "0.0.0.0",
+    "java": {
+      "vendor": "Eclipse Adoptium",
+      "version": "17.0.4.1"
+    },
+    "kubernetes": {
+      "detected": "false"
+    },
+    "library": {
+      "name": "unknown",
+      "version": "unknown"
+    },
+    "location": {
+      "country": "PT"
+    },
+    "os": {
+      "name": "Mac OS X",
+      "os_arch": "aarch64",
+      "version": "13.3.1"
+    },
+    "quarkus": {
+      "version": "999-SNAPSHOT"
+    },
+    "timezone": "Europe/Lisbon"
+  },
+  "event": "PROD_BUILD",
+  "integrations": {},
+  "messageId": "api-2PCI9LWQM8oRyHIS6fCQMIQ0dlR",
+  "originalTimestamp": "2023-05-01T15:21:22.266033Z",
+  "properties": {
+    "app_extensions": [
+      {
+        "artifact_id": "quarkus-resteasy-jackson",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-smallrye-openapi",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-hibernate-orm",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-opentelemetry",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-jdbc-postgresql",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-arc",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      }
+    ]
+  },
+  "receivedAt": "2023-05-01T15:21:23.184Z",
+  "timestamp": "2023-05-01T15:21:22.266Z",
+  "type": "track",
+  "userId": "872332b4-4b21-42ee-854b-d86c8629e78b"
+}
+----
+
+== Where are the config files?
+Analytics configurations are stored at the `~/redhat` folder:
```

Comment:
```suggestion
Analytics configurations are stored at the `~/.redhat` folder:
```

Reply:
Will be Windows users aware what `~` means?

Reply:
Maybe ...`.redhat folder in the user's home directory`?

Reply:
Sure. Will do.